src/library.js

```diff
@@ -1521,31 +1521,11 @@ LibraryManager.library = {
   gnu_dev_minor: 'minor',
 
   // ==========================================================================
-  // setjmp.h
+  // setjmp / longjmp support
   // ==========================================================================
 
-  longjmp__sig: 'vii',
-#if SUPPORT_LONGJMP
-  longjmp: function(env, value) {
-    {{{ from64('env') }}};
-    _setThrew(env, value || 1);
-    throw 'longjmp';
-  },
-#else
-  longjmp__deps: [function() {
-    error('longjmp support was disabled (SUPPORT_LONGJMP=0), but it is required by the code (either set SUPPORT_LONGJMP=1, or remove uses of it in the project)');
-  }],
-  // will never be emitted, as the dep errors at compile time
-  longjmp: function(env, value) {
-    abort('longjmp not supported');
-  },
-#endif
```

Comment:
Maybe keep the `#if !SUPPORT_LONGJMP` stubs around for when folks build with `SUPPORT_LONGJMP=0`?

Reply:
Done